### PR TITLE
feat: Update cakeVault contract address and ABI

### PR DESCRIPTION
**Pull request by ChefMomota**

### CODE REVIEW

File: src/config/abi/cakeVaultV2.json [hunk 1]
```diff
@@ -430,7 +523,7 @@
       { "internalType": "uint256", "name": "lastUserActionTime", "type": "uint256" },
       { "internalType": "uint256", "name": "lockStartTime", "type": "uint256" },
       { "internalType": "uint256", "name": "lockEndTime", "type": "uint256" },
-      { "internalType": "uint256", "name": "boostDebt", "type": "uint256" },
```
Comment (ChefMomota) [inline src/config/abi/cakeVaultV2.json:433]: `boostDebt` changed to `userBoostedShare`

File: src/config/abi/cakeVaultV2.json [hunk 2]
```diff
@@ -430,7 +523,7 @@
...
+      { "internalType": "uint256", "name": "userBoostedShare", "type": "uint256" },
```
Comment (ChefMomota) [inline src/config/abi/cakeVaultV2.json:526]: boostDebt changed to userBoostedShare

Comment (ChefMomota) [inline src/config/abi/cakeVaultV2.json:526]: `boostDebt` changed to `userBoostedShare`

**Comment by 0xjojoex** [comment]
let me pull the fake cake out of the old pool first :D

File: src/config/abi/cakeVaultV2.json (hunk 2 above)
Comment (ChefMomota) [reply src/config/abi/cakeVaultV2.json:526]: @chef-jojo new comment

**Review by 0xjojoex — APPROVED** [review]

**Pull request merged**